src/napari_matplotlib/base.py

```diff
@@ -69,25 +73,55 @@ def add_single_axes(self) -> None:
         self.axes = self.figure.subplots()
         self.apply_napari_colorscheme(self.axes)
 
-    @staticmethod
-    def apply_napari_colorscheme(ax: Axes) -> None:
+    def apply_napari_colorscheme(self, ax: Axes) -> None:
         """Apply napari-compatible colorscheme to an Axes."""
+        # get the foreground colours from current theme
+        theme = napari.utils.theme.get_theme(self.viewer.theme, as_dict=False)
+        fg = theme.foreground.as_hex()  # fg is a muted contrast to bg
```

Comment:
`text_color` and `fg_color` would be slightly more readable variable names in my opinion, but that's not a strong opinion.

Reply:
Nono. You may have.